cloud/amazon/ec2_elb_lb.py

```diff
@@ -142,6 +143,21 @@
     required: false
     default: 60
     version_added: "2.1"
+  tags:
+    description: the tags dict to apply
+    type: dict
+    required: false
+    sample: { "Department": "Information Retrieval" }
+    version_added: "2.1"
+  purge_tags:
```

Comment:
append_tags might be a better name as it will keep consistent with other implementations of similar options (i.e. append in user module)